test_apps/npm/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "license": "UNLICENSED",
+  "private": true,
+  "scripts": {
+    "encore": "encore"
```

Comment:
I just exposed `encore` as a script, otherwise npm (through `npm run encore`) won't find `encore` executable.

I did the same for pnpm/yarn pnp for consistency purpose.